### ISSUE TITLE
print function not working when inside another function.

### ISSUE THREAD

**Issue by Highly-Unstable-Hadron** [Venger-King/WaveGrass#6]
I'm running the source code of the Beta version:
```
define test(){
    a = prompt("Type something: ")
    print(a)
}

a = prompt("Type something: ")
print(a)

test()
```
The print function seems to output my input fine when outside the function test, and the function test asks for input with the prompt but doesn't print it.
![Screenshot 2022-03-30 124633](https://user-images.githubusercontent.com/102580192/160773740-2426f8b0-25eb-4b4d-9ceb-418951e85ae9.jpg)

**Comment by Venger-King** [comment]
This would be fixed in the next release

**Comment by Venger-King** [comment]
Fixed in v0.1.0-beta
